go/test/endtoend/onlineddl/vrepl/onlineddl_vrepl_test.go

```diff
@@ -400,11 +398,11 @@ func TestSchemaChange(t *testing.T) {
 		// to be strictly higher than started_timestamp
 		assert.GreaterOrEqual(t, lastThrottledTimestamp, startedTimestamp)
 		component := row.AsString("component_throttled", "")
-		assert.Contains(t, []string{string(vreplication.VCopierComponentName), string(vreplication.VPlayerComponentName)}, component)
+		assert.Contains(t, []string{throttlerapp.VCopierName, throttlerapp.VPlayerName}, component)
```

Comment:
IMO it might be worth making a new type that's an alias for string so that what the string represents is even more explicit. This make it clear to future code writers that we're not using arbitrary strings.

Reply:
You're right. I initially did that but then that affected so many lines of code, casting into and out of string, that I decided to hold off. I can still do that for this PR if you think it's worjthwhile.

Reply:
I'll leave it up to you. I think it's probably worth it, but we can always do it later too.

Reply:
Done in https://github.com/vitessio/vitess/pull/13195/commits/37a7c2fd92b1b8e18bdf34eab9dfd60caaf0faa8. I'm not sure how I feel about it, what do you think?